rpc/src/eth_rpc.rs

```diff
@@ -144,8 +127,20 @@ impl EthApiServer for EthApi {
         )
         .await
         .map_err(|err| RpcServeError::Message(err.to_string()))?;
-        let output = result_and_state.result.into_output().unwrap_or_default();
-        Ok(output)
```

Comment:
*nit* I think this change is clearer if it goes into its own commit. I was trying to figure out how it was related for a bit, until I went back to the PR summary to realize that it wasn't 😆 .